reg/hoangstark.dat

```diff
@@ -0,0 +1,2 @@
+SHA-256:d00f8c8456b30305c9b8bbc40165327a8da02e634b44dc6162c82bc2ee4eba81
```

Comment:
You should test your algo with my ID and make sure it matches before applying for your ID ;)